docs/csharp/tutorials/default-interface-methods-versions.md

```diff
@@ -45,7 +45,7 @@ The team agreed on the most likely default implementation: a loyalty discount fo
 
 The upgrade should provide the functionality to set two properties: the number of orders needed to be eligible for the discount, and the percentage of the discount. This makes it a perfect scenario for default interface methods. You can add a method to the `ICustomer` interface, and provide the most likely implementation. All existing, and any new implementations can use the default implementation, or provide their own.
 
-First, add the new method to the implementation:
+First, add the new method to the interface, including the implementation:
```

Comment:
To me this sounds like add the method to the interface and then a class that implements the interface. Would it be better if it was something closer to 
```suggestion
First, add the new method to the interface, including the body of the method:
```